lib/text/ttml_text_parser.js

```diff
@@ -202,6 +202,12 @@ shaka.text.TtmlTextParser = class {
     }
     goog.asserts.assert(cueElement, 'cueElement should be non-null!');
 
+    let spaceStyle = null;
```

Comment:
nit: let spaceStyle = cueElement.getAttribute...

Reply:
Yes after this solution moves as expected to the right.

Reply:
style addressed in a subsequent commit